src/components/organisms/PaneManager/PaneManagerLeftMenu.tsx

```diff
@@ -196,7 +188,7 @@ const PaneManagerLeftMenu: React.FC = () => {
         placement="right"
       >
         <MenuButton
-          isSelected={Boolean(activeProject) && !leftDrawerVisible && leftMenuSelection === 'templates-pane'}
+          isSelected={Boolean(activeProject) && leftMenuSelection === 'templates-pane'}
```

Comment:
For the sake of simplicity and core readability can we create a function which receives leftMenuSelection value as a param and we check the active project and === between current value and selected value. It seems that all of the are the same more or less